src/index.html

```diff
@@ -7,8 +7,48 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="MOYO Інтернет-магазин">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
```

Comment:
```suggestion
              class="nav_link"
```
Consider fixing indentations according to the code style guide